packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -427,6 +440,14 @@ export default class OperatorModeStackItem extends Component<Signature> {
     this.containerEl.scrollTop = 0;
   });
 
+  private doCloseAnimation = task(async () => {
```

Comment:
let's use a `dropTask` since if this is performed multiple times you don't want to rerun the subsequent performs